apps/platform-app/src/app/[locale]/(auth)/(routes)/signin/(components)/LoginDev/LoginDev.tsx

```diff
@@ -1,16 +1,30 @@
+import { I18nNamespaces } from '@/i18n/request'
+import { useTranslations } from 'next-intl'
+
 // components
 import { GithubLoginButton } from '@/app/[locale]/(auth)/(components)/GithubLoginButton/GithubLoginButton'
-import { I18nNamespaces } from '@/i18n/request'
+import { LinkedInLoginButton } from '@/app/[locale]/(auth)/(components)/LinkedInLoginButton/LinkedInLoginButton'
+import { Roles } from '@/app/[locale]/(auth)/_models/User.model'
 import { Box } from '@gdh/ui-system'
-import { useTranslations } from 'next-intl'
+import styles from '../../page.module.scss'
 
 const LoginDev = () => {
   const t = useTranslations(I18nNamespaces.LoginDev)
   return (
     <Box>
       <h2>{t('title')}</h2>
       <p>{t('description')}</p>
-      <GithubLoginButton label={t('loginWithGithub')} />
+      <div className={styles.btnsContainer}>
```

Comment:
**praise**: Nice use of a container for login buttons, improving layout and user experience.